File: pytools/examples/csharp/client/Protocol/AckChatSendOk.cs
using System.Collections;
using System.Collections.Generic;


public class AckChatSendOk
{
	private byte _channel;
	private uint _uid;
	private string _uname;
	private string _content;


	public AckChatSendOk(Packet packet)
	{
		this._channel = packet.ReadByte();
		this._uid = packet.ReadUint();
		this._uname = packet.ReadString();
		this._content = packet.ReadString();
	}


	public byte channel
	{
		get { return this._channel; }
		set { this._channel = value; }
	}

	public uint uid
	{
		get { return this._uid; }
		set { this._uid = value; }
	}

	public string uname
	{
		get { return this._uname; }
		set { this._uname = value; }
	}

	public string content
	{
		get { return this._content; }
		set { this._content = value; }
	}

}

File: pytools/examples/java/client/protocol/AckChatSendOk.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class AckChatSendOk
{
	private byte channel;
	private int uid;
	private String uname;
	private String content;


	public AckChatSendOk(Packet packet)
	{
		channel = packet.readByte();
		uid = packet.readInt();
		uname = packet.readString();
		content = packet.readString();
	}



	public int getChannel()
	{
		return PacketUtil.readUByte(this.channel);
	}

	public long getUid()
	{
		return PacketUtil.readUInt(this.uid);
	}

	public String getUname()
	{
		return this.uname;
	}

	public String getContent()
	{
		return this.content;
	}

}
